src/components/RichTextEditor/plugins/heading/headingPlugin.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2025-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { headingPlugin as _headingPlugin, PARAGRAPH_ELEMENT_TYPE, toggleHeading } from "@ndla/editor";
+import { isHotkey } from "is-hotkey";
+import { Transforms } from "slate";
+import { getHotKey } from "./headingUtils";
+
+export const headingPlugin = _headingPlugin.configure({
+  shortcuts: {
+    toggleNormalText: {
+      keyCondition: isHotkey(getHotKey("normal-text")),
+      handler: (editor, event, logger) => {
+        if (!editor.selection) return false;
+        event.preventDefault();
+        logger.log("Toggling normal text");
+        Transforms.setNodes(editor, { type: PARAGRAPH_ELEMENT_TYPE }, { at: editor.selection });
+        return true;
+      },
+    },
+    toggleH1: {
```

Comment:
Skulle denne vore toggleH2? Og tilsvarende for den under?

Reply:
ups